test/js/bun/shell/commands/yes.test.ts

```diff
@@ -0,0 +1,24 @@
+import { $, ShellOutput } from "bun";
+import { describe, test, expect, beforeEach } from "bun:test";
+
+$.throws(false);
+
+describe("yes", async () => {
```

Comment:
what a weird command, it just keeps printing the same value over and over again?

Reply:
yes